Bad import request (POST) shuts down server

Running release 1.2.81 (844012864ff4a7ecb5b236983578d3e36b915304) 
After creating a new pad, execute the following curl command:

```
curl -v --form press=OK 127.0.0.1:9001/p/<Your pad id>/import
```

Causes the server to exit with console error:

```
[2013-03-13 12:53:55.409] [ERROR] console - TypeError: Cannot read property 'stack' of null
    at exports.doImport.async.series.knownFileEndings (...../git/ether_latest/src/node/handler/ImportHandler.js:63:49)
    at IncomingForm.parse (...../git/ether_latest/src/node_modules/formidable/lib/incoming_form.js:117:9)
    at IncomingForm.EventEmitter.emit (events.js:90:17)
    at IncomingForm._maybeEnd (...../git/ether_latest/src/node_modules/formidable/lib/incoming_form.js:377:8)
    at MultipartParser.IncomingForm._initMultipart.parser.onEnd (...../git/ether_latest/src/node_modules/formidable/lib/incoming_form.js:320:10)
    at MultipartParser.write.callback (...../git/ether_latest/src/node_modules/formidable/lib/multipart_parser.js:102:31)
    at MultipartParser.write (...../git/ether_latest/src/node_modules/formidable/lib/multipart_parser.js:259:15)
    at IncomingForm.write (...../git/ether_latest/src/node_modules/formidable/lib/incoming_form.js:139:34)
    at IncomingMessage.<anonymous> (...../git/ether_latest/src/node_modules/formidable/lib/incoming_form.js:91:12)
    at IncomingMessage.EventEmitter.emit (events.js:93:17)
```

I produced this issue while attempting to test importing files, obviously it isn't a proper import, but I don't think it should shut down the server.

--- a/src/node/handler/ImportHandler.js
+++ b/src/node/handler/ImportHandler.js
@@ -60,7 +60,7 @@ exports.doImport = function(req, res, padId)
       form.parse(req, function(err, fields, files) { 
         //the upload failed, stop at this point
         if(err || files.file === undefined) {
-          console.warn("Uploading Error: " + err.stack);
+          if(err) console.warn("Uploading Error: " + err.stack);
           callback("uploadFailed");
         }
         //everything ok, continue

